modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -658,12 +658,11 @@ public void registerObjectRelationshipsRelatedInfoCollectionProviders(
 				objectDefinition1.getObjectDefinitionId());
 
 		for (ObjectRelationship objectRelationship : objectRelationships) {
-			if (!Objects.equals(
-					objectRelationship.getType(),
-					ObjectRelationshipConstants.TYPE_MANY_TO_MANY) &&
-				!Objects.equals(
-					objectRelationship.getType(),
-					ObjectRelationshipConstants.TYPE_ONE_TO_MANY)) {
+			Set<String> defaultObjectRelationshipTypes =
+				ObjectRelationshipUtil.getDefaultObjectRelationshipTypes();
```

Comment:
We can create this variable before the FOR because it will always be the same value.

Reply:
If we create a method, this will be unnecessary.